source/contributing.md

```diff
@@ -0,0 +1,174 @@
+
+7. Add the changed files.
+
+  ```sh
+   git add <changed files>
+  ```
+
+8. Commit your changes
+
+  ```sh
+   git commit -m 'Fixed typing error in <document name>'
+  ```
+
+9. Open a pull request to Percona
+
+  ```sh
+   git push <my repo> <my_changes>
+  ```
+
+### Building the documentation
+
+To verify how your changes look, generate the static site with the documentation. This process is called _building_. You can use the following methods:
+
+- [Use Docker](#use-docker)
+- [Install Sphinx and build locally](#install-sphinx-and-build-locally)
+
+#### Using Docker
+
+1. [Get Docker](https://docs.docker.com/get-docker/)
+2. We use [this Docker image](https://hub.docker.com/r/ddidier/sphinx-doc) to build documentation. Run the following command:
+
+```
+docker run --rm -i -v `pwd`:/doc -e USER_ID=$UID ddidier/sphinx-doc:0.9.0 make clean html
+```
+
+If Docker can't find the image locally, it first downloads the image, and then runs it to build the documentation.
+
+1. Go to the `doc/build/html` directory and open the `index.html` file to see the documentation.
+2. Your static site will look different from the one on the web site. We use a Percona theme that is rendered when the documentation is published to the website. You can add the following [sphinx-doc options](https://www.sphinx-doc.org/en/master/man/sphinx-build.html) to view the documentation with the [Alabaster](https://alabaster.readthedocs.io/en/latest/) theme
+
+  - `-b` selects the builder. This example uses HTML.
+  - `-D` overrides a selection in conf.py. The example phrase changes the html_theme to `alabaster`.
+
+    The following example creates a HTML build with the Alabaster theme.
+
+    ```sh
+      docker run --rm -i -v `pwd`:/doc -e USER_ID=$UID ddidier/sphinx-doc:0.9.0 sphinx-build -b html -D 'html_theme=alabaster' source build/html
+    ```
+
+3. To create a PDF version of the documentation, run the following command:
+
+  ```sh
+   docker run -i -v `pwd`:/doc -e USER_ID=$UID ddidier/sphinx-doc:0.9.0 make clean latex && docker run -i -v `pwd`:/doc -e USER_ID=$UID ddidier/sphinx-doc:0.9.0 make clean latexpdf
+  ```
+
+The PDF document is in the `doc/build/latex` folder.
+
+#### Installing Sphinx and building locally
+
+1. Install [pip](https://pip.pypa.io/en/stable/installing/)
+2. Install [Sphinx](https://www.sphinx-doc.org/en/master/usage/installation.html).
+3. While in the root directory of the doc project, run the following command to build the documentation:
+
+  ```sh
+   make clean html
+  ```
+
+This command does two tasks. The `clean` command rebuilds the `/build` directory and the `latexpdf` commands creates a PDF document. The PDF document is stored in the `/build/html` folder.
```

Comment:
This sentence is out of place. It tells about PDFs in the middle of the build html section.

Reply:
Fixed.